worker/db.go

```diff
@@ -0,0 +1,38 @@
+package worker
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/mini-maxit/worker/utils"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+type postgresDataBase struct {
+	Db *gorm.DB
+}
+
+func Connect(db * postgresDataBase) *gorm.DB {
+	return db.Db
+}
+
+// Connect to the database using GORM
+func NewPostgresDatabase() *postgresDataBase {
+	Config := LoadConfig()
```

Comment:
We should load config only once, for example in the main function, and then pass this config to all the "NewSomething" functions. Example [here](https://github.com/HermanPlay/WebApp/blob/master/backend/internal/api/http/initialization.go)